cmd/jobbuzz-api/main.go

```diff
@@ -27,6 +51,11 @@ func main() {
 
 	r := gin.Default()
 
+	if gin.IsDebugging() {
+	    r.GET("/", playgroundHandler())
+	}
+	r.POST("/graphql", graphqlHandler(&service))
```

Comment:
we're not putting the graphql endpoint within the /api/v1 route group?

Reply:
I looked up the best practice for graphql, they do not recommend versioning graphql api because there should be no need to when you can specify what type of data to request and your api should be backwards compatible.

Reply:
makes sense!